scripts/import/probabilities/marsh_probabilities.py

```diff
@@ -0,0 +1,320 @@
+    "loss_of_function_mp" : 64,
+    "dominant_negative_mp" : 65
+}
+
+def get_details_from_file(file):
+    """
+        Extracts lines from a file, skipping the header.
+
+        This function reads the contents of a file, skipping the first line
+        (assumed to be a header) and returning the remaining lines.
+
+        Args:
+            file (str): The path to the file to be read.
+
+        Returns:
+            None: This function does not return a value, but it stores the
+            lines after the header in the variable `skip_header`.
+
+        Example:
+            get_details_from_file("data.txt")
+    """
+    with open(file, "r") as opened_file:
+        lines = opened_file.readlines()
+        skip_header = lines[1:]
+
+    
+    return skip_header
+
+def get_locus_id_from_g2p_db(list_lines, host, port, db, password, user):
+    """
+        Retrieves locus IDs from a gene-to-phenotype (G2P) database and appends them to the input list.
+
+        This function connects to a MySQL database and executes a query to fetch the locus ID
+        for each gene symbol (first item in each line) from the 'locus' table. It appends the
+        fetched locus ID to the corresponding line in `list_lines`. If a locus ID is not found,
+        it appends `None`.
+
+        Args:
+            list_lines (list of str): A list of strings or lists, where each element contains
+                                  a gene symbol as the first item.
+            host (str): The database host address.
+            port (int): The port number to connect to the database.
+            db (str): The name of the database.
+            password (str): The password for authenticating with the database.
+            user (str): The username for authenticating with the database.
+
+        Returns:
+            list: The modified `list_lines` with locus IDs appended to each line.
+              If no locus ID is found for a gene symbol, `None` is appended.
+
+        Example:
+            list_lines = ["BRCA1 info", "TP53 info"]
+            updated_lines = get_locus_id_from_g2p_db(list_lines, "localhost", 3306, "g2p_db", "password", "user")
+    """
+
+    get_locus_source = """ SELECT id from locus where name = %s
+"""
+
+    database = MySQLdb.connect(host=host,port=port,user=user,passwd=password,db=db)
+    cursor = database.cursor()
+
+    for i, line in enumerate(list_lines):
+        if isinstance(line, str):
+            line = line.split()
+            gene_symbol = line[0]
+            cursor.execute(get_locus_source, (gene_symbol,))
+            locus_id = cursor.fetchone()
+    
+
+            if locus_id:
+                print(locus_id)
+                line.append(locus_id[0])
+            else:
+                line.append(None)
+            
+            list_lines[i] = line
+        
+    cursor.close()
+    database.close()
+
+
+    return list_lines
+
+
+def get_hgnc_id_from_g2p_db(list_lines, host, port, db, password, user):
```

Comment:
I don't think this subroutine is necessary.
The script already linked the score to the gene ID, this subroutine is only using that existing link to create a link to the HGNC.

Reply:
Yeah i agree, removed HGNC (locus_identifier) from the gene_stats table